teku/src/main/java/tech/pegasys/teku/cli/options/ValidatorClientOptions.java

```diff
@@ -49,16 +49,29 @@ public class ValidatorClientOptions {
       ValidatorConfig.DEFAULT_FAILOVERS_SEND_SUBNET_SUBSCRIPTIONS_ENABLED;
 
   @CommandLine.Option(
-      names = {"--Xbeacon-node-event-stream-syncing-status-query-period"},
+      names = {"--Xbeacon-nodes-syncing-status-query-period"},
```

Comment:
do we need millis here? I would have thought seconds should be plenty..

Reply:
Actually agree, I did it for the tests, but in the acceptance tests we can just use one second since they usually take bit longer anyways